src/scenes/Event/components/EventTxHistory/TxRow.js

```diff
@@ -26,31 +24,17 @@ export default class TxRow extends Component {
   }
 
   toggle = () => {
-    // probably need some logic here for not expanding when clicking on
+    // TODO: probably need some logic here for not expanding when clicking on
```

Comment:
create new issue, I think it deserves a issue

Reply:
I think it already has an issue. I think Harrison is actually working on this right now